packages/core/parcel-bundler/src/assets/JSAsset.js

```diff
@@ -76,6 +77,10 @@ class JSAsset extends Asset {
   }
 
   async pretransform() {
+    if (WORKER_RE.test(this.contents) && WORKERMODULE_RE.test(this.contents)) {
+      this.contents = this.contents.replace(WORKERMODULE_RE, "type: 'classic'");
```

Comment:
can you do this inside the AST visitor rather than as a string replacement? it will be much more reliable that way